core/src/main/java/hudson/PluginManager.java

```diff
@@ -813,7 +813,7 @@ public boolean accept(File dir, String name) {
                         return false;
                     }
                 });
-                LOGGER.log(INFO, "Upgraded detached plugins (and dependencies): {0}",
+                LOGGER.log(FINEST, "Upgraded detached plugins (and dependencies): {0}",
```

Comment:
This an important debugging help and shouldn't be lower than the default visible INFO.